multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/listeners/EnterTestingPhaseListenerTest.java

```diff
@@ -0,0 +1,100 @@
+package org.cloudfoundry.multiapps.controller.process.listeners;
+
+import org.cloudfoundry.multiapps.controller.api.model.ImmutableOperation;
+import org.cloudfoundry.multiapps.controller.api.model.Operation;
+import org.cloudfoundry.multiapps.controller.core.util.ApplicationConfiguration;
+import org.cloudfoundry.multiapps.controller.persistence.query.impl.OperationQueryImpl;
+import org.cloudfoundry.multiapps.controller.persistence.services.HistoricOperationEventService;
+import org.cloudfoundry.multiapps.controller.persistence.services.OperationService;
+import org.cloudfoundry.multiapps.controller.persistence.services.ProcessLoggerProvider;
+import org.cloudfoundry.multiapps.controller.persistence.services.ProcessLogsPersister;
+import org.cloudfoundry.multiapps.controller.persistence.services.ProgressMessageService;
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.cloudfoundry.multiapps.controller.process.flowable.FlowableFacade;
+import org.cloudfoundry.multiapps.controller.process.util.MockDelegateExecution;
+import org.cloudfoundry.multiapps.controller.process.util.StepLogger;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+class EnterTestingPhaseListenerTest {
+
+    // needed because of @InjectMocks
+    @Mock
+    private ProgressMessageService progressMessageService;
+    // needed because of @InjectMocks
+    @Mock
+    private ProcessLoggerProvider processLoggerProvider;
+    // needed because of @InjectMocks
+    @Mock
+    private ProcessLogsPersister processLogsPersister;
+    // needed because of @InjectMocks
+    @Mock
+    private HistoricOperationEventService historicOperationEventService;
+    // needed because of @InjectMocks
+    @Mock
+    private FlowableFacade flowableFacade;
+    // needed because of @InjectMocks
+    @Mock
+    private ApplicationConfiguration configuration;
+
+    @Mock
+    private OperationService operationService;
+    @Mock
+    private OperationQueryImpl operationQuery;
+    @Mock
+    private StepLogger.Factory stepLoggerFactory;
+    @Mock
+    private StepLogger stepLogger;
+
+    @InjectMocks
+    private EnterTestingPhaseListener listener = new EnterTestingPhaseListener();
```

Comment:
As far as I remember, you don't need to create new object when you use `@InjectMocks` annotation